pybamm/solvers/base_solver.py

```diff
@@ -613,6 +629,13 @@ def solve(self, model, t_eval=None, external_variables=None, inputs=None):
             pybamm.logger.info(
                 "Discontinuity events found at t = {}".format(discontinuities)
             )
+            if isinstance(inputs, list):
```

Comment:
For now, it is not possible to solve for several sets of inputs parameters and have discontinuities.
this is because discontinuities where computed only once (using the first element of `inputs_list`) and setting the restart state after a discontinuity isn't obvious since `model.y0` only contains a single state (see below)